build/database/tests/test_generate_database.py

```diff
@@ -1,63 +0,0 @@
-# ############################################################################### #
-# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
-#
-# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI
-# SPDX - License - Identifier: GPL-3.0-or-later
-# ############################################################################### #
-"""
-Tests for code that generates a working local database for autoredction
-"""
-import os
-import unittest
-
-from mock import Mock
-from sqlalchemy.exc import OperationalError
-
-from build.database.generate_database import (get_test_user_sql, get_sql_from_file,
-                                              run_sql)
-from build.utils.common import ROOT_DIR
-
-
-# pylint:disable=missing-docstring
-def raise_operational_error(_):
-    raise OperationalError(statement='Fail',
-                           params=None,
-                           orig=None)
-
-
-# pylint:disable=missing-docstring
-class TestGenerateDatabase(unittest.TestCase):
```

Comment:
The functions tested here have been deleted as they're no longer used and the one that's left over in generate_database.py doesn't have a test